container-storage-setup.sh

```diff
@@ -416,7 +416,7 @@ systemd_escaped_filename () {
 remove_systemd_mount_target () {
   local mp=$1
   local filename=$(systemd_escaped_filename $mp)
-  if [ -f /etc/systemd/system/$filename ];then
+  if [ -f /etc/systemd/system/$filename -a -x /usr/bin/systemctl ];then
```

Comment:
Fixed to still remove the file, although this is never going to happen in the real world.